### PR TITLE
Update audit_report.py

### PR DESCRIPTION
**Pull request by AkeAmadi**

### CODE REVIEW

**Comment by LisaBChen** [comment]
ok, I tested your code, and it only prints out this: 
image
Could you send me a screenshot of what you got when you tested this?

**Comment by LisaBChen** [comment]
and when I directly tested your method in audit_report.py , it just resulted in this, instead of an actual gpa number: 
image

**Comment by LisaBChen** [comment]
If it's easier on you, for the "core gpa" row, "elective gpa" row, and "combined gpa" row (not the "outstanding requirements" section), in the audit report word doc, you can just have the functions ``` get_core_gpa_section(), get_elective_gpa_section(), and get_combined_gpa_section() ``` return a float instead of a string. I can just access those functions individually, and assign them to separate variables. In addition, I can just have the bolded text labels for those 3 rows (for example " **Core GPA: ** ", " **Elective GPA: ** ", and " ** Combined GPA: ** ") be handled separately from your code. 
image

File: audit_report.py [hunk 1]
```diff
@@ -1,11 +1,49 @@
...
+    def get_gpas_section(self):
+        gpas_section = ""
+        gpas_section += self.get_core_gpa_section() + "\n"
+        gpas_section += self.get_elective_gpa_section() + "\n"
+        gpas_section += self.get_combined_gpa_section() + "\n"
+        return gpas_section
+        
+    def get_core_gpa_section(self):
+	    core_gpa_section = "Core GPA: "
+	    core_gpa = str(self.get_core_gpa)
```
Comment (sujayvadlakonda) [inline audit_report.py:31]: The error that Lisa is getting is occurring bc line 31 should be 
```
core_gpa = str(self.get_core_gpa())
```
Fix these everywhere

**Review by sujayvadlakonda — CHANGES_REQUESTED** [review]

**Comment by LisaBChen** [comment]
You're using the outdated version of both our branches code.

**Comment by LisaBChen** [comment]
just letting you know know that I'm uploading your additions to the code for you.

**Comment by LisaBChen** [comment]
akelanda, did you upload your latest changes, to the outstanding requirements related code, to any of the branches yet? Because your outstanding requirements code still has that bug/error we mentioned before.

**Review by LisaBChen — CHANGES_REQUESTED** [review]
Where are the changes you made to fix the outstanding requirements section error in your code? Because I remember you saying before that you don't upload your latest code to the github repo.

**Comment by LisaBChen** [comment]
akelanda, your code for the outstanding requirements section of the audit, still have compilation errors in it etc for some of the transcripts, and it's causing bugs for the rest of program.

**Pull request closed without merging**